release/k8s_tests/run_gcs_ft_on_k8s.py

```diff
@@ -207,6 +207,22 @@ def start_sending_traffics(duration, users):
     )
     print(helm_install_logs)
 
+    timeout_wait_for_locust_s = 300
+    while timeout_wait_for_locust_s > 0:
+        labels = [
+            f"app.kubernetes.io/instance=ray-locust-{cluster_id}",
+            "app.kubernetes.io/name=locust,component=master",
+        ]
+        pods = cli.list_namespaced_pod("default", label_selector=",".join(labels))
+        assert len(pods.items) == 1
+
+        if pods.items[0].status.phase == "Pending":
+            print("Wait for locust pod is ready...")
+            time.sleep(30)
+            timeout_wait_for_locust_s -= 30
+        else:
```

Comment:
When other state, it will fail under the port forwarding eventually. 
In the port forwarding step, it will complain something like
`error: unable to forward port because pod is not running. Current status=Pending` 
So i don't bother to check other status.